src/sched/affinity.c

```diff
@@ -13,8 +14,8 @@ int sched_setaffinity(pid_t tid, size_t size, const cpu_set_t *set)
 
 int pthread_setaffinity_np(pthread_t td, size_t size, const cpu_set_t *set)
 {
-	struct lthread *lt = (struct lthread *)td;
-	lt->attr.state |= BIT(LT_ST_PINNED);
+	// struct lthread *lt = (struct lthread *)td;
+	// lt->attr.state |= BIT(LT_ST_PINNED);
 	/* Don't call sched_setaffinity because ethreads are
 	   already pinned to cores in the starter */
```

Comment:
This comment is a bit wrong, because it's ignoring lthreads.  I added `sched_setaffinity` to the set of syscalls that are silently ignored (locally, I need to submit the PR), so this can be reverted to do the syscall.